File: python/pangram.py
def is_pangram(n:str) -> bool:
    if len(n) <= 5:
        raise ValueError("Invalid input string")
    
    # set in python only hold unqiue value and has many function
    alphabet = set("abcdefghijklmnopqrstuvwxyz")
    n = set(n.lower())
    
    for ch in alphabet:
        if ch not in n:
            return False
  
    return True


def main():
    sentence = "the quick brown fox jumps over the lazy dog"

    isPangram = is_pangram(sentence)

    print(f"The sentence is: {isPangram}")


if __name__ == "__main__":
    main()

File: javascript/pangram.js
function isPangram(word){
    let word_str = String(word)

    word_str = word_str.split("")

    const letter = ['a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z']
    for(let char of letter){
        if(!word_str.includes(char)){
            return false
        }
    }

    return true
}

function main(){
    const sentences = "The quick brown fox jumps over the lazy dog"

    check = isPangram(sentences)
    console.log(`The ${sentences} is pangram: ${check}`)
}

main()